halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/ci/gcb/GoogleCloudBuildAccount.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2019 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.halyard.config.model.v1.ci.gcb;
+
+import com.netflix.spinnaker.halyard.config.model.v1.node.CIAccount;
+import com.netflix.spinnaker.halyard.config.model.v1.node.LocalFile;
+import com.netflix.spinnaker.halyard.config.model.v1.node.SecretFile;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper = false)
+public class GoogleCloudBuildAccount extends CIAccount {
+  private String name;
+  private String project;
+  private String subscriptionName;
+  @LocalFile @SecretFile private String jsonKey;
```

Comment:
Hi! I was wondering why this was `jsonKey` instead of `jsonPath`? It would seem all of the other google accounts prefer `jsonPath`.